scripts/validate-brazil-config

```diff
@@ -0,0 +1,109 @@
+#!/usr/bin/env python3
+from pathlib import Path
+import json
+import sys
+import re
+
+# This script validates that:
+#  - All SDK modules are mapped or skipped for internal import
+#  - All non-SDK dependencies of imported modules are mapped to internal
+#    packages
+#
+# Usage: validate-brazil-config [module-paths-file] [dependencies-file]
+
+# Generating module-paths-file:
+#   mvn exec:exec -Dexec.executable=pwd -pl \!:aws-sdk-java-pom,\!:sdk-benchmarks,\!:module-path-tests -q 2>&1 > modules.txt
+#
+#   Generates contents similar to:
+#     /workspace/aws-sdk-java-v2/build-tools
+#     /workspace/aws-sdk-java-v2/core
+#     /workspace/aws-sdk-java-v2/core/annotations
+#     /workspace/aws-sdk-java-v2/utils
+#     /workspace/aws-sdk-java-v2/test/test-utils
+#     /workspace/aws-sdk-java-v2/core/metrics-spi
+#     /workspace/aws-sdk-java-v2/http-client-spi
+#     /workspace/aws-sdk-java-v2/core/endpoints-spi
+#     /workspace/aws-sdk-java-v2/core/identity-spi
+#     /workspace/aws-sdk-java-v2/core/http-auth-spi
+#     ...
+
+# Generating dependencies-file:
+#   mvn dependency:list -DexcludeTransitive=true -DincludeScope=runtime 2>&1 > deps.txt
+#
+#   Generates content similar to:
+#
+#   [INFO] -----------------< software.amazon.awssdk:test-utils >------------------
+#   [INFO] Building AWS Java SDK :: Test :: Test Utils 2.31.61-SNAPSHOT     [6/493]
+#   [INFO]   from test/test-utils/pom.xml
+#   [INFO] --------------------------------[ jar ]---------------------------------
+#   [INFO]
+#   [INFO] --- dependency:3.1.1:list (default-cli) @ test-utils ---
+#   [INFO]
+#   [INFO] The following files have been resolved:
+#   [INFO]    org.slf4j:slf4j-api:jar:1.7.36:compile -- module org.slf4j [auto]
+#   [INFO]    org.junit.jupiter:junit-jupiter:jar:5.10.0:compile -- module org.junit.jupiter
+#   [INFO]    com.fasterxml.jackson.core:jackson-core:jar:2.15.2:compile -- module com.fasterxml.jackson.core
+#   ...
+
+brazil_import_config_path=".brazil.json"
+
+with open(brazil_import_config_path) as f:
+  brazil_import_config = json.loads(f.read())
+
+modules_path=sys.argv[1]
+
+service_modules = set()
```

Comment:
You are creating `service_modules` and adding to this set in line 64, but not doing anything with it after. There is only core module validation, is this intentional? 

Is this related to the 
`"services": { "skipImport": true }` part in the `.brazil.json` file somehow?

Reply:
Good point, I'll remove it